templates/admin/pprof.tmpl

```diff
@@ -0,0 +1,85 @@
+<h4 class="ui top attached header">
+	{{.locale.Tr "admin.monitor.pprof"}}
+</h4>
+<div class="ui attached segment">
+	<h4>{{.locale.Tr "admin.monitor.pprof.cpuprofile"}}</h4>
+	<form action="{{AppSubUrl}}/admin/monitor/cpu-profile">
+		<div class="ui form">
+			<div class="field">
+				<label for="pprof-cpuprofile-duration">{{.locale.Tr "admin.monitor.pprof.duration"}}</label>
+				<input id="pprof-cpuprofile-duration" type="text" name="duration" placeholder="{{.locale.Tr "admin.monitor.pprof.duration_placeholder"}}">
+			</div>
+			<input class="ui button basic" type="submit" value="{{.locale.Tr "admin.monitor.pprof.download"}}">
+		</div>
+	</form>
+	<h4>{{.locale.Tr "admin.monitor.pprof.fgprof"}}</h4>
+	<form action="{{AppSubUrl}}/admin/monitor/fgprof">
+		<div class="ui form">
+			<div class="field">
+				<label for="pprof-fgprof-duration">{{.locale.Tr "admin.monitor.pprof.duration"}}</label>
+				<input id="pprof-fgprof-duration" type="text" name="duration" placeholder="{{.locale.Tr "admin.monitor.pprof.duration_placeholder"}}">
+			</div>
+			<div class="field">
+				<label for="pprof-fgprof-format">{{.locale.Tr "admin.monitor.pprof.fgprof.format"}}</label>
+				<select class="ui search dropdown" id="pprof-fgprof-format" name="format">
+					<option value="pprof" selected>pprof</option>
+					<option value="folded">folded</option>
+				</select>
+			</div>
+			<input class="ui button basic" type="submit" value="{{.locale.Tr "admin.monitor.pprof.download"}}">
+		</div>
+	</form>
+	<h4>{{.locale.Tr "admin.monitor.stacktrace"}}</h4>
+	<form action="{{AppSubUrl}}/admin/monitor/stacktrace-profile">
+		<div class="ui form">
+			<div class="inline field">
+				<div class="ui checkbox">
+					<input id="pprof-stacktrace-flat" type="checkbox" tabindex="0" class="hidden" name="flat">
```

Comment:
`tabindex=0`?
Why should you not be able to focus this checkbox?
And why hidden?
See
```suggestion
					<input id="pprof-stacktrace-flat" type="checkbox" class="hidden" name="flat">
```
or even
```suggestion
					<input id="pprof-stacktrace-flat" type="checkbox" name="flat">
```

Reply:
They're fomantic checkboxes - this is a hidden thing that fomantic will restyle